projects/igniteui-angular/src/lib/services/overlay/overlay.ts

```diff
@@ -187,20 +277,10 @@ export class IgxOverlayService implements OnDestroy {
         if (info.settings.positionStrategy.settings.openAnimation) {
             this.playOpenAnimation(info);
         } else {
+            //  to eliminate flickering show the element just before onOpened fire
+            info.elementRef.nativeElement.parentElement.visibility = null;
```

Comment:
Do we need to set it to null or delete will work too?

Reply:
delete will not work here, as per my tests :)